webapp/channels/src/components/suggestion/at_mention_provider/at_mention_provider.tsx

```diff
@@ -218,7 +266,7 @@ export default class AtMentionProvider extends Provider {
         const priorityProfilesIds = {};
         const priorityProfiles = this.filterPriorityProfiles();
 
-        priorityProfiles.forEach((member) => {
+        priorityProfiles.forEach((member: UserProfile) => {
             priorityProfilesIds[member.id] = true;
```

Comment:
There's an error in this line for `priorityProfilesIds[member.id]`
```
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{}'.
No index signature with a parameter of type 'string' was found on type '{}'.
```

Reply:
The reason for the error is that type of `priorityProfilesIds` is not set and the initial value is an empty object `{}` so TypeScript is not sure if key should be a string. 

Try changing `const priorityProfilesIds = {};` to `const priorityProfilesIds: {[key: string]: boolean} = {};`